docs/installation/config/env_config.rst

```diff
@@ -0,0 +1,284 @@
+  Note that if you're on Google Cloud, sending e-mail via port 25 is completely
+  blocked and you should use 487 for TLS.
+
+* ``EMAIL_HOST_USER``: username to connect to the mail server. Default empty.
+
+* ``EMAIL_HOST_PASSWORD``: password to connect to the mail server. Default empty.
+
+* ``EMAIL_USE_TLS``: whether to use TLS or not to connect to the mail server.
+  Defaults to ``False``. Should be ``True`` if you're changing the ``EMAIL_PORT`` to
+  ``487``.
+
+* ``MIN_UPLOAD_SIZE``: the max allowed size of POST bodies, in bytes. Defaults to
+  4GiB. Note that you should also configure your web server to allow this.
+
+* ``DOCUMENTEN_UPLOAD_CHUNK_SIZE``: chunk size in bytes for large file uploads -
+  determines the size for a single upload chunk. Defaults to the value of
+  ``MIN_UPLOAD_SIZE``. Note that making this larger than ``MIN_UPLOAD_SIZE`` breaks large
+  file uploads.
+
+* ``DOCUMENTEN_UPLOAD_READ_CHUNK``: chunk size in bytes for large file uploads - when
+  merging upload chunks, this determines the number of bytes read to copy to the
+  destination file. Defaults to 6 MiB.
+
+* ``SENDFILE_BACKEND``: which backend to use for authorization-secured upload
+  downloads. Defaults to ``sendfile.backends.nginx``. See
+  [django-sendfile2](https://pypi.org/project/django-sendfile2/) for available
+  backends.
+
+* ``SENTRY_DSN``: URL of the sentry project to send error reports to. Default
+  empty, i.e. -> no monitoring set up. Highly recommended to configure this.
+
+* ``JWT_EXPIRY``: duration a JWT is considered to be valid, in seconds. Defaults to 3600 -
+  1 hour.
+
+* ``JWT_LEEWAY``: JWT validation has a time aspect, usually in the form of the ``iat`` and
+  ``nbf`` claims. Clock drift between server and client can occur. This setting allows
+  specifying the leeway in seconds, and defaults to ``0`` (no leeway). It is advised to
+  not make this larger than a couple of minutes.
+
+* ``LOG_STDOUT``: whether to log to stdout or not. For Docker environments, defaults to
+  ``True``, for other environments the default is to log to file.
+
+* ``LOG_QUERIES``: enable (query) logging at the database backend level. Note that you
+  must also set ``DEBUG=1``, which should be done very sparingly!
+
+* ``LOG_LEVEL``: control the verbosity of logging output. Available values are ``CRITICAL``,
+  ``ERROR``, ``WARNING``, ``INFO`` and ``DEBUG``. Defaults to ``WARNING``.
+
+* ``LOG_REQUESTS``: enable logging of the outgoing requests. Defaults to ``False``.
+
+* ``PROFILE``: whether to enable profiling-tooling or not. Applies to the development
+  settings only. Defaults to ``False``.
+
+* ``CMIS_ENABLED``: whether to enable the CMIS adapter. Defaults to ``False``.
+
+* ``CMIS_MAPPER_FILE``: name of the file containing the mapping between the Django and Document Management System names
+    for document properties. See the installation section for more details.
+    Defaults to the absolute path of ``open-zaak/config/cmis_mapper.json``.
```

Comment:
Indentation needs to be reduced here, otherwise the formatting is a bit strange:
![image](https://github.com/open-zaak/open-zaak/assets/29249171/ae88f18d-3ecd-4336-bc87-3cbeeabda4ef)

```suggestion
* ``CMIS_MAPPER_FILE``: name of the file containing the mapping between the Django and Document Management System names
  for document properties. See the installation section for more details.
  Defaults to the absolute path of ``open-zaak/config/cmis_mapper.json``.
```